ga4gh/frontend.py

```diff
@@ -203,6 +203,9 @@ def configure(configFile=None, baseConfig="ProductionConfig", extraConfig={}):
     if configFile is not None:
         app.config.from_pyfile(configFile)
     app.config.update(extraConfig.items())
+    # Setup file handle cache max size
+    ga4gh.datamodel.fileHandleCache.setMaxCacheSize(
```

Comment:
Change this to `datamodel.fileHandle...`, and add an import at the top for `import ga4gh.datamodel as datamodel`.